exercise/templates/exercise/staff/_submissions_table.html

```diff
@@ -124,3 +140,54 @@
 </table>
 
 {% include "exercise/staff/_regrade_submissions.html" %}
+<script>
```

Comment:
This JS can definitely be moved to its own file since there's no django rendering involved. So move this to `exercise/static/exercise/filter_submissions_by_tag.js` or such, and then load here like 

```
<script src="{% static 'exercise/filter_submissions_by_tag.js' %}"></script>
```

Reply:
Moved to own file